controllers/user.js

```diff
@@ -1,15 +1,36 @@
 const User = require('../models/user');
+const Exercise = require('../models/workoutlog');
+
 
 function index(req, res, next) {
-    User.find({}, function(err, user) {
-      res.render('user/index', { user,
-      user: req.user });
-    });
-  }
+ const user = req.user 
+    User.find({_id:req.user._id}, function(err, user) {
+      console.log(user)
+       Exercise.find({}, function(err, exercise) {
+      console.log("exercise", exercise)
+      res.render('user/index', { user, exercise });
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7828699)
```suggestion
      res.render("user/index", { user, exercise });
```